xbmc/platform/posix/filesystem/SMBWSDiscoveryListener.cpp

```diff
@@ -119,6 +120,33 @@ static const std::array<std::pair<std::string, std::string>, 1> address_tag{
 static const std::array<std::pair<std::string, std::string>, 1> types_tag{
     {{"<wsd:Types>", "</wsd:Types>"}}};
 
+static const std::string get_msg =
+    "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
+    "<soap:Envelope "
+    "xmlns:pnpx=\"http://schemas.microsoft.com/windows/pnpx/2005/10\" "
+    "xmlns:pub=\"http://schemas.microsoft.com/windows/pub/2005/07\" "
+    "xmlns:soap=\"http://www.w3.org/2003/05/soap-envelope\" "
+    "xmlns:wsa=\"http://schemas.xmlsoap.org/ws/2004/08/addressing\" "
+    "xmlns:wsd=\"http://schemas.xmlsoap.org/ws/2005/04/discovery\" "
+    "xmlns:wsdp=\"http://schemas.xmlsoap.org/ws/2006/02/devprof\" "
+    "xmlns:wsx=\"http://schemas.xmlsoap.org/ws/2004/09/mex\"> "
```

Comment:
I always appreciate when line lists are organized alphabetically

Reply:
I don't get it. "xmlns:pnpx" ... "xmlns:wsx" are sorted.

Reply:
I'm saying that I appreciate that you sorted